### DISCUSSION TITLE
I think is a bug here?

### DISCUSSION THREAD

**Discussion by edicoderr** [EXERLOG/exer_log#103] [Q&A] [answered]
![Screenshot from 2022-08-20 08-48-18](https://user-images.githubusercontent.com/111631581/185739861-0cc679f8-7b1e-4885-b896-25ee176c6f1a.png)

Why i can see the app? A Bug?

**Comment by jorre127** [comment, marked as answer]
You’re running the app in chrome, but currently web isn’t supported. The other error message you see is because not every package is updated to work with flutter 3.0 yet
